std/experimental/allocator/common.d

```diff
@@ -190,18 +190,17 @@ package size_t divideRoundUp(size_t a, size_t b)
 /**
 Returns `s` rounded up to a multiple of `base`.
 */
-@nogc nothrow pure
+@nogc nothrow pure @trusted
```

Comment:
> you can make it trusted if it takes ubyte[]

As far as I see, that doesn't affect safety. Slicing a `void[]` is safe, isn't it? You might get garbage when you cast it to something else, but that's always unsafe. dmd thinks it's safe.

But rounding up `s.ptr` without doing bounds checking seems unsafe. When rounding puts `p` beyond `s`'s bounds, then `end < p`, and `end - p` wraps around.